lib/nodes/div.js

```diff
@@ -2,9 +2,15 @@ const PREFIX = '\n';
 const SUFFIX = '';
 
 module.exports = (props, children) => {
+    // If a node has children: For each string node, add a prefix and suffix.
+    if (children.length > 0) {
+        return children.map(node => (typeof node === 'string' ? (PREFIX + node + SUFFIX) : node));
```

Comment:
Your prettier passed it as well as eslint.    If it isn't clear though, I can switch to an if statement instead.